src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -120,110 +128,63 @@ const formConfig = {
   subTitle: 'Form 21-526EZ',
   // getHelp: GetFormHelp, // TODO: May need updated form help content
   chapters: {
-    reviewVeteranDetails: {
-      title: (isReviewPage) => {
-        const baseString = 'Review Veteran Details';
-        if (isReviewPage) {
-          return baseString.replace('Review ', '');
-        }
-        return baseString;
-      },
+    veteranDetails: {
+      title: (isReviewPage) => `${isReviewPage ? 'Review ' : ''}Veteran Details`,
```

Comment:
I'm not sure that this should say any more than `Veteran Details`.

Reply:
Perhaps a question for @goldenmeanie as that is how the prototype is set up?